compiler/rustc_middle/src/mir/syntax.rs

```diff
@@ -126,7 +126,13 @@ pub enum RuntimePhase {
     /// Beginning with this phase, the following variant is disallowed:
     /// * [`ProjectionElem::Deref`] of `Box`
     PostCleanup = 1,
-    Optimized = 2,
+    /// Beginning with this phase, the following variant is disallowed:
+    /// * [`TerminatorKind::Yield`]
+    /// * [`TerminatorKind::GeneratorDrop`]
+    ///
+    /// Furthermore, `Copy` operands are allowed for non-`Copy` types.
+    GeneratorsLowered = 2,
+    Optimized = 3,
```

Comment:
This needs a doc comment (that's pre-existing though, so probably material for a different PR).